src/compile/selection/interval.ts

```diff
@@ -118,14 +118,22 @@ const interval:SelectionCompiler = {
       name: undefined,
       type: 'rect',
       encode: {
-        enter: {fill: {value: '#eee'}},
+        enter: {
+          fill: {value: '#333'},
+          fillOpacity: {value: 0.125},
+          stroke: undefined
+        },
         update: update
       }
     }].concat(marks, {
       name: name + BRUSH,
       type: 'rect',
       encode: {
-        enter: {fill: {value: 'transparent'}},
+        enter: {
```

Comment:
To ensure that the fill color of the brush doesn't interfere with the marks/grid lines/etc. but still allow users to interact with the brush. Will add a comment.